### PR TITLE
Jenkins enable rover SITL test

### PR DESCRIPTION
**Pull request by dagar**
**Describe problem solved by the proposed pull request**
There are no existing SITL tests for the rover.

**Describe your preferred solution**
A rover mission, `rover_mission_1`, was added to Jenkins. It is a simple mission consisting of driving to 3 waypoints.

### CODE REVIEW

**Comment by ItsTimmy** [comment]
I am currently trying to get this to work, but it will depend upon this PR in the sitl_gazebo repo: https://github.com/PX4/sitl_gazebo/pull/329

File: .ci/Jenkinsfile-SITL_tests_ASan [hunk 1]
```diff
@@ -61,6 +61,13 @@ pipeline {
               vehicle: "iris"
             ],
 
+            [
+              name: "Rover 1",
```
Comment (julianoes) [inline .ci/Jenkinsfile-SITL_tests_ASan]: @dagar what was your thought behind "Rover 1"? Is there going to be another one? :smile:

Comment (dagar) [reply .ci/Jenkinsfile-SITL_tests_ASan]: Wishful thinking that we were going to have significantly more of these tests.

**Comment by dagar** [comment]
Thanks for finishing this @ItsTimmy.

**Pull request merged**